package.json

```diff
@@ -27,7 +27,7 @@
     "browser"
   ],
   "optionalDependencies": {
-    "default-browser-id": "1.0.2"
+    "default-browser-id": "1.0.4"
```

Comment:
Cool. Added a commit to change all the dependencies to use `^`, which is what npm uses by default now.